spec/lib/authorities/linked_data/find_term_spec.rb

```diff
@@ -156,6 +161,15 @@
             .to eq ['headings beginning with the word [Scientific;] and subdivision [Science] under ethnic groups and individual wars, e.g. [World War, 1939-1945--Science]']
           expect(results['predicates']['http://www.w3.org/2004/02/skos/core#inScheme']).to eq ['http://id.loc.gov/authorities/subjects']
         end
+
+        it 'has correct primary predicate values for second request' do
+          expect(results[:uri]).to eq 'http://id.loc.gov/authorities/subjects/sh85118553'
+          expect(second_results[:uri]).to eq 'http://id.loc.gov/authorities/subjects/sh1234'
```

Comment:
This test fails before the fix.  The failure is only seen on the second find by id which is why this test checks `results` before checking `second_results`.